### PR TITLE
Add other example distros

### PR DESCRIPTION
**Pull request by davidcassany**
This PR adds a Fedora based image (blue) and an Ubuntu based image (orange)

### CODE REVIEW

**Comment by frelon** [comment]
Awesome work 🚀

**Comment by davidcassany** [comment]
Ok this last commit makes the CI to react on `arm64` and `all-distros` labels. If `arm64` is added it will trigger, in addition, tests for `aarch64` builds. If the label `all-distros` is included it will trigger all examples in the CI. If both labels are present only SUSE distributions are tested on aarch64 (no `blue` nor `orange`).

Full suite of tests is only run for `green` (openSUSE Leap) flavor.

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
   pull_request:
+    types: [ unlabeled, labeled, opened, synchronize, reopened ]
```
Comment (kkaempf) [inline .github/workflows/pr.yaml:7]: Does this run on PRs without any labels now ? 😨

File: tests/smoke/smoke_test.go [hunk 2]
```diff
@@ -100,11 +100,12 @@ var _ = Describe("Elemental Smoke tests", func() {
 			Expect(out).Should(ContainSubstring("UTC"))
 		})
 
-		It("has default localectl configuration from cloud-init", func() {
+		// locale setting doesn't work on Fedora ¯\_(ツ)_/¯
+		/*It("has default localectl configuration from cloud-init", func() {
```
Comment (kkaempf) [inline tests/smoke/smoke_test.go:104]: So a bug in Fedora makes us disable this test for all other distros ? 🤔

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (davidcassany) [reply .github/workflows/pr.yaml:7]: yes, these are the events that actually trigger the run. So on opening, synchronizing, labeling, unlabeling, etc.

File: tests/smoke/smoke_test.go (hunk 2 above)
Comment (davidcassany) [reply tests/smoke/smoke_test.go:104]: Not sure if a bug in Fedora... but yes that can be improved

Comment (davidcassany) [reply tests/smoke/smoke_test.go]: Re-enabled

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (davidcassany) [reply .github/workflows/pr.yaml:7]: Without any label runs the exact same thing that used to run, but this PR adds the functionality that when the `all-distro` label is around in addition it also runs the CI for other distros. Running `arm64` tests by including the `arm64` label was already possible. 

The change here on this line only express under which events the workflow is re-triggered. The default value is `opened`, `synchronize` and  `reopened`. Then I added `labeled` and `unlabeled` so it also reacts on label changes and there is no need to add the desired labels before hand at PR creation.

I think probably a better approach would be moving the `arm64` tests out of the PR scope and run them only on demand by manual workflow and/or based on some schedule, but not on PRs.

**Pull request merged**